src/components/spaces/breeding/breeding-view.tsx

```diff
@@ -128,19 +138,36 @@ export class BreedingView extends BaseComponent<IProps, IState> {
                 litters.slice().reverse().map((litter, i) => (
                   <div className="litter" key={"litter" + label + (litters.length - i)}>
                     {
-                      litter.map((org, j) => (
-                        <StackedOrganism
-                          key={"org" + j}
-                          organism={org}
-                          organismImages={[org.nestImage]}
-                          height={60}
-                          showSelection={false}
-                          showSex={breeding.showSexStack}
-                          showHetero={breeding.showHeteroStack}
-                          showLabel={breeding.showGametes}
-                          isOffspring={true}
-                        />
-                      ))
+                      litter.map((org, j) => {
+                        const litterNum = litters.length - i;
+                        return (
+                          <div
+                            className="offspring-container"
+                            onMouseEnter={currentLitter === (litterNum - 1)
+                                          ? this.handleOffspringHoverEnter(j)
+                                          : undefined}
+                            onMouseLeave={currentLitter === (litterNum - 1)
+                                          ? this.handleOffspringHoverExit(j)
+                                          : undefined}
+                            key={"org-cont" + j}>
+                            <StackedOrganism
+                              key={"org" + j}
+                              organism={org}
+                              organismImages={[org.nestImage]}
+                              height={60}
+                              showSelection={false}
+                              showGameteSelectionOnHover={breeding.showGametes && (currentLitter === (litterNum - 1))}
```

Comment:
I find the combination of `showGameteSelectionOnHover` and `showGameteSelection` to be a little confusing, because the hover state is something this container view already knows about, because if we're hovering we must have this org equal to `this.state.offspringHightlightIndex`.

I feel like `showGameteSelectionOnHover` is redundant. It adds a class that has an associated `:hover` state in the css, but that rule is always trumped by the addition of the `show` class which is passed down because of `showGameteSelection`, and which is always there if we are hovering.

I might be just confused, though.

Reply:
Good call.  My goal was to allow a caller to set the green gamete circle to be hoverable or to be shown due to some other interaction (which we need since a user can, for example, hover over a gamete icon which needs to turn the green gamete circle on) - hence the two values passed via props.  However, you're right, at present we have everything in the container state to effectively remove `showGameteSelectionOnHover` from the props and handle all of the green gamete circle states simply with `showGameteSelection`.

Anyway, I pushed a change to remove `showGameteSelectionOnHover`